Patches/Card_AttachAbilities_NewSpecialAbilities.cs

```diff
@@ -21,13 +21,14 @@ public static bool Prefix(CardInfo info, Card __instance)
 			if (NewSpecialAbility.specialAbilities.Exists(ability =>
 				info.specialAbilities.Contains(ability.specialTriggeredAbility)))
```

Comment:
When referencing the special abilities of a CardInfo, the property 'SpecialAbilities' (capitalized) is the correct property to use because it will get all abilities on the base card as well as those from all card modifications. This happens multiple times in this patch.